app/graph/graphql_crud_operations.rb

```diff
@@ -418,64 +400,81 @@ def self.define_annotation_type(type, fields = {}, &block)
+
+      field :annotated, AnnotatedUnion, 'Annotated item'
+
+      connection :annotations, -> { AnnotationUnion.connection_type }, 'Annotations describing this annotation' do
+        argument :annotation_type, !types.String, 'Comma-separated list of annotation types' # TODO Convert to array of strings
+
+        resolve ->(annotation, args, _ctx) {
+          Annotation.where(annotation_type: args['annotation_type'].split(',').map(&:strip), annotated_type: ['Annotation', annotation.annotation_type.camelize], annotated_id: annotation.id)
         }
       end
-      instance_exec :annotator, AnnotatorType, &GraphqlCrudOperations.annotation_fields
-      instance_exec :version, VersionType, &GraphqlCrudOperations.annotation_fields
 
-      connection :assignments, -> { UserType.connection_type } do
+      connection :assignments, -> { UserType.connection_type }, 'Users assigned to this annotation' do
         resolve ->(annotation, _args, _ctx) {
           annotation.assigned_users
         }
       end
 
-      connection :annotations, -> { AnnotationUnion.connection_type } do
-        argument :annotation_type, !types.String
-        resolve ->(annotation, args, _ctx) {
-          Annotation.where(annotation_type: args['annotation_type'], annotated_type: ['Annotation', annotation.annotation_type.camelize], annotated_id: annotation.id)
+      connection :attribution, -> { UserType.connection_type }, 'Users who have contributed to this annotation' do
+        resolve ->(annotation, _args, _ctx) {
+          ids = annotation.attribution.split(',').map(&:to_i)
+          User.where(id: ids)
         }
       end
 
-      field :locked, types.Boolean
+      field :field, types.String, 'Value of annotation field' do # TODO Map field type
+        argument :name, !types.String, 'Field name'
+
+        resolve ->(annotation, args, _ctx) {
+          annotation.get_field_value(args['name'])
+        }
+      end
 
+      # TODO Remove
       field :project, ProjectType
 
       field :image_data, JsonStringType
 
-      field :data, JsonStringType
+      field :data, JsonStringType, 'Annotation content'
 
       field :parsed_fragment, JsonStringType
 
       instance_eval(&block) if block_given?
-    end
-  end
 
-  def self.annotation_fields
-    proc do |name, field_type = types.String, method = nil|
-      field name do
-        type field_type
+      field :locked, types.Boolean, 'Is annotation locked from further modifications?'
 
-        resolve -> (annotation, _args, _ctx) {
-          annotation.send(method.blank? ? name : method)
+      field :dbid, types.Int, 'Database id of this record'
+      field :created_at, types.String, 'Datetime of annotation creation' do resolve -> (annotation, _args, _ctx) { annotation.created_at.to_i.to_s } end
+      field :updated_at, types.String, 'Datetime of annotation last update' do resolve -> (annotation, _args, _ctx) { annotation.updated_at.to_i.to_s } end
```

Comment:
Same (at least flag with a `TODO` for now): timestamps should be converted to integers